.idea/misc.xml

```diff
@@ -3,7 +3,7 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="11" project-jdk-type="JavaSDK">
```

Comment:
Wait, I forgot to mention. Can you please revert the changes to this file?

Reply:
I have reverted it, unexpected changes while opening the project in IntelliJ with configured SDK